src/go/api/vm/util.go

```diff
@@ -172,3 +172,10 @@ func (this cancelableReader) Read(p []byte) (int, error) {
 		return this.r.Read(p)
 	}
 }
+
+func getTimestamp() string {
+
+	refTime := time.Now()
+	return fmt.Sprintf("%d%02d%02d_%02d%02d", refTime.Year(), refTime.Month(),
+		refTime.Day(), refTime.Hour(), refTime.Minute())
```

Comment:
@eric-c-wood for consistency's sake, I'd prefer to see the `time.Format` function used here. To reproduce the same timestamp you're producing here, you could use `refTime.Format("20060102_1500")`.

Reply:
@activeshadow will fix.